src/main/java/net/sacredlabyrinth/phaed/simpleclans/Helper.java

```diff
@@ -930,12 +932,14 @@ public static String formatMaxInactiveDays(int max) {
     @NotNull
     public static String getFormattedClanStatus(Clan clan, CommandSender sender) {
         ArrayList<String> statuses = new ArrayList<>();
-        if (clan.isPermanent()) statuses.add(SimpleClans.lang("permanent", sender));
+        if (clan.isPermanent()) {
+            statuses.add(lang("permanent", sender));
+        }
         if (clan.isVerified()) {
-            statuses.add(SimpleClans.lang("verified", sender));
+            statuses.add(lang("verified", sender));
         } else {
-            statuses.add(SimpleClans.lang("unverified", sender));
+            statuses.add(lang("unverified", sender));
         }
-        return statuses.stream().filter(Objects::nonNull).collect(Collectors.joining(", "));
```

Comment:
filter is used for this place: [bonk](https://github.com/Tomut0/SimpleClans/commit/5b51f74de80c95e55151c1b54a7e4a8ea47b982d#diff-c116b099e86ebc72d4a8ec5171ccf08d294e8eae36df7c546fcd90a014733cbeR109)